testcases/extensions-map.http

```diff
@@ -0,0 +1,10 @@
+POST / HTTP/1.1
+Host: handler.example.com
+Content-Length: 0
+ce-specversion: 0.4-wip
+ce-id: 123
+ce-source: https://cloudevents.io/extensions-map
+ce-type: io.cloudevents.extensions-map-test
+ce-this0is0ok-": ok
+ce-this0is0ok-: {\"u005c": 17}
```

Comment:
I believe the `\` needs to go inside the double-quotes

Reply:
Removed.